profiling/tests/phpt/phpinfo_01.phpt

```diff
@@ -46,9 +46,9 @@ assert(isset($values["Version"]));
 // Check exact values for this set
 $sections = [
     ["Profiling Enabled", "false"],
-    ["Experimental CPU Time Profiling Enabled", "true"],
-    ["Allocation Profiling Enabled", "true"],
-    ["Endpoint Collection Enabled", "true"],
+    ["Experimental CPU Time Profiling Enabled", "false"],
+    ["Allocation Profiling Enabled", "false"],
+    ["Endpoint Collection Enabled", "false"],
```

Comment:
Think we should try to enhance these messages? "disabled because profiling as a whole is disabled" or something?

Reply:
I went with a short version: `false (profiling disabled)`